wiki/Do_you_really_want_to_ask_peppy/ro.md

```diff
@@ -0,0 +1,15 @@
+# Chiar dorești să-l întrebi pe peppy?
+
+Salutare! peppy înțelege dorința ta de a dori să-l contactezi personal, dar te roagă să înțelegi mai întâi că acesta **nu** este locul unde vrei să-i ceri ajutor sau sprijin. El alege să păstrează asistența osu! separat de mesajele personale și de asemenea, consideră că este destul de greu să răspundă la PM-urile de pe forum. Prin urmare, vă solicită să faceți una dintre următoarele opțiuni:
+
+## Suportul tehnic
+
+Majoritatea problemelor tehnice pot fi rezolvate căutând sau postând în secțiunea [Forum-ul de ajutor](https://osu.ppy.sh/community/forums/5). Sunt mulți alții acolo care te pot ajuta cu problemele tale, așa că vei găsi o soluție mai repede decât să-l contactezi pe el.
+
+## Suport general osu!
+
+Pentru probleme legate de conturi, probleme cu plățile, urmărirea interdicțiilor, întrebări despre turnee și **și orice altceva legat de având nevoie de ajutor pe osu!**, vă rugăm să contactați [echipa de suport](/wiki/People/The_Team/Account_support_team) prin trimiterea unui e-mail la adresa [accounts@ppy.sh](mailto:accounts@ppy.sh) (interogări de cont) sau [osu@ppy.sh](mailto:osu@ppy.sh) (orice altceva).
```

Comment:
"Pentru probleme legate de conturi, probleme cu plățile, detalii despre ban-uri, întrebări legate de turnee **și orice altceva în caz de nevoie de ajutor pe osu!**, vă rugăm să contactați [echipa de suport](/wiki/People/The_Team/Account_support_team) prin trimiterea unui e-mail la adresa [accounts@ppy.sh](mailto:accounts@ppy.sh) (interogări de cont) sau [osu@ppy.sh](mailto:osu@ppy.sh) (orice altceva)."

Nu știu sigur daca "ban" trebuie tradus neapărat ca și "interdicție" în acest context.

Reply:
pe crowdin e folosit acelasi termen de interdictie https://crowdin.com/translate/osu-web/all/en-ro?filter=basic&value=0#q=ban

dar alt cuvant sau exprimare nu am nici eu.

Reply:
termenul de interdictie a mai fost folosit in trecut pentru acest scop deci personal zic ca este ok. dau mark ca si resolved